components/Toolbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Remove this file from your PR - you have to get your Toolbar PR merged in before incorporating this code